application/classes/Ushahidi/Repository/Post/Tags.php

```diff
@@ -0,0 +1,60 @@
+<?php defined('SYSPATH') OR die('No direct access allowed.');
+
+/**
+ * Ushahidi Post Relation Repository
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+class Ushahidi_Repository_Post_Tags extends Ushahidi_Repository_Post_Value
```

Comment:
@Angamanga have a look at this. This allows us to have an attribute `type` that loads from the posts_tags table ... I've just added a form_attribute_id to the existing table. Still working on changes to the posts repo though